docs/pages/blog/docs-updates.md

```diff
@@ -0,0 +1,55 @@
+As we plan to release new products that are outside of the scope of Material UI, it'd be confusing to keep everything under the same roof.
+That's why, from now on, each product will have its own documentation, to clarify the boundaries between them and to facilitate navigation if your interest lies in one specific product.
+
+## Wait... what MUI products are you talking about?
+
+Currently, our main two products are:
+
+- **MUI Core:** a collection of libraries that offers you foundational components, which includes Material UI, MUI Base (unstyled components), MUI System, and more. It really is a product line.
+- **MUI X:** a collection of advanced components, meant primarily to support data-heavy applications. Differently from Core, each component lives under its own package.
+
+## What has changed?
+
+All MUI products are still under the [mui.com](http://mui.com) domain but each of them now has an isolated URL and, therefore, documentation.
+We added an identifier and menu at the top left of the docs to ease navigation:
+
+<img src="/static/blog/docs-updates/docs-separation.png" style="width: 796px; margin-top: 16px; margin-bottom: 16px;" alt="Screen shot of the product identifier menu" />
+
+As for the URLs, this is how they look now:
+
+- MUI Core:
+  - Material UI: [https://mui.com/material-ui/\_](https://mui.com/material-ui/getting-started/installation/)
+  - MUI Base: [https://mui.com/base/\_](https://mui.com/base/getting-started/installation/)
+  - MUI System: [https://mui.com/system/\_](https://mui.com/system/basics/)
+- MUI X:
+  - Data grid: [https://mui.com/x/react-data-grid/\_](https://mui.com/x/react-data-grid/)
+  - Date and time pickers: [https://mui.com/x/react-date-pickers/\_](https://mui.com/x/react-date-pickers/getting-started/)
+
+## What do I gain?
+
+We believe separating the documentation will significantly improve your navigation experience.
+This rings especially true for MUI Base and MUI X contributors and customers, considering that we have reworked how search works.
+Now searching for any Data Grid specific article, for example, is much faster.
+
+## Moving forward
+
+This separation will become increasingly beneficial as each product grows, such as additional components in MUI X and MUI Base.
+With it, comes more content to share and an even greater need for better space to accommodate it.
+
+It will also help with the slight confusion we caused by introducing a different design to our marketing website, especially considering that our only design system package so far is Material UI.
+However, as we move forward working on [the second design system package](https://deploy-preview-30686--material-ui.netlify.app/experiments/), the separation will allow each to use their default styles, which will certainly help to avoid confusion about what the component looks like by installing the library, as it was when v4's docs used Material Design without any modification.
```

Comment:
I would cut the mention of the past confusion and instead clarify what's meant by "each using their default styles."

```suggestion
As we move forward working on [our second design system package](https://deploy-preview-30686--material-ui.netlify.app/experiments/), the separation will make it possible for us to build the documentation for each system using its default styles—so the docs themselves will serve as examples of their corresponding component libraries.
```